### PR TITLE
Add Support for Oracle DBMS

### PR DESCRIPTION
**Pull request by yangdsh**
Support Oracle.

### CODE REVIEW

File: client/controller/config/sample_postgres_config.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
```
Comment (bohanjason) [inline client/controller/config/sample_postgres_config.json]: You may not want to change this file.

File: client/driver/controller.log [hunk 2]
```diff
@@ -0,0 +1,16 @@
+:compileJava
```
Comment (bohanjason) [inline client/driver/controller.log]: remove this file

File: client/driver/next_config [hunk 3]
```diff
@@ -0,0 +1 @@
+{"recommendation": {"checkpoint_timeout": "13min", "shared_buffers": "1GB", "temp_buffers": "846MB", "effective_cache_size": "1GB", "random_page_cost": 5.986029151137736, "work_mem": "464MB", "default_statistics_target": 1440, "max_wal_size": "855MB", "bgwriter_lru_maxpages": 7, "checkpoint_completion_target": 0.8045125117003064, "max_parallel_workers_per_gather": 0}}
```
Comment (bohanjason) [inline client/driver/next_config]: remove this file

File: client/driver/oltp.log [hunk 4]
```diff
@@ -0,0 +1,22 @@
+02:27:54,087 (DBWorkload.java:262) INFO  - ======================================================================
```
Comment (bohanjason) [inline client/driver/oltp.log]: remove this file

File: client/driver/driver_config.json [hunk 5]
```diff
@@ -1,20 +1,20 @@
...
-  "database_type" : "postgres",
-  "database_name" : "tpcc",
-  "database_disk": "/dev/xvda1",
-  "database_conf": "/etc/postgresql/9.6/main/postgresql.conf",
-  "database_save_path": "/home/ubuntu/ottertune",
-  "username" : "bohan",
-  "password" : "bohan",
-  "oltpbench_home": "/home/ubuntu/oltpbench",
-  "oltpbench_config": "/home/ubuntu/oltpbench/config/tpcc_config_postgres.xml",
+  "database_type" : "oracle",
```
Comment (bohanjason) [inline client/driver/driver_config.json]: do not change this file

File: client/driver/knobs/postgres-96.json [hunk 6]
```diff
@@ -2,17 +2,17 @@
     {
         "name": "effective_cache_size",
```
Comment (bohanjason) [inline client/driver/knobs/postgres-96.json]: do not change this file

**Comment by dvanaken** [comment]
@yangdsh please also remove any changes you made to the existing code/configs for tuning Postgres. If you found a bug in the Postgres code then you should submit the fix(es) in a separate PR.

**Comment by dvanaken** [comment]
See #201

**Pull request closed without merging**